### PR TITLE
Added a chess board to the UI

### PR DESCRIPTION
**Pull request by arezic00**

### CODE REVIEW

File: app/src/main/java/com/example/cheatchess/ChessView.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class ChessView(context: Context?, attrs: AttributeSet?) : View(context, attrs) {
+    private val paint = Paint()
+    private val rectSide = 130f
+    private val marginLeft = 20f
+    private val marginTop = 200f
+
+    override fun onDraw(canvas: Canvas?) {
+        super.onDraw(canvas)
+        for (i in 0..7)
```
Comment (mulhollandaf) [inline app/src/main/java/com/example/cheatchess/ChessView.kt:18]: I would make this more clear by using x and y.

File: app/src/main/java/com/example/cheatchess/ChessView.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            for (j in 0..7) {
+                paint.color = if ((i + j) % 2 == 0) Color.LTGRAY else Color.GRAY
+                canvas?.drawRect(marginLeft + rectSide*j,marginTop + rectSide*i,marginLeft + rectSide*(j+1),marginTop + rectSide*(i+1),paint)
```
Comment (mulhollandaf) [inline app/src/main/java/com/example/cheatchess/ChessView.kt:21]: I think you could make this more clear with better formatting.

File: app/src/main/java/com/example/cheatchess/ChessView.kt (hunk 1 above)
Comment (arezic00) [reply app/src/main/java/com/example/cheatchess/ChessView.kt]: I used row and col

File: app/src/main/java/com/example/cheatchess/ChessView.kt (hunk 2 above)
Comment (arezic00) [reply app/src/main/java/com/example/cheatchess/ChessView.kt]: I split the starting and ending positions of the rectangle in separate lines.
the naming of the parameters is kind of unintuitive, it should be:
-startX
-startY
-endX
-endY
-paint

**Review by mulhollandaf — APPROVED** [review]
One more thing you may want to consider is named parameters. 
drawRect (
startX =..,
startY = ...
)

This is especially helpful for parameters that are the same type, to make sure you don't mix them up, by accident.

**Pull request merged**